### ISSUE TITLE
Trying to use hook(useState) set value and onchange will lose focus

### ISSUE THREAD

**Issue by Kseng** [RIP21/react-simplemde-editor#150]
I trying to use hook (useState) to set value and onchange value, 
but always lose the focus, when key in something.

thanks.

```
const [markdownValue, setMarkdownValue] = useState("");

const handleChange = value => {
    setMarkdownValue(value);
};

<SimpleMDEReact value={value} onChange={handleChange}  />

```

**Comment by RIP21** [comment]
This has been fixed in `5.0.0` please update and note breaking changes.

**Comment by Kseng** [comment]
@RIP21 Great , thanks a lot of.
